packages/core/src/components/ToastContainer/ToastContainer.stories.mdx

```diff
@@ -82,17 +83,19 @@ export const App: React.FC = () => {
     </Story>
 </Preview>
 
-### Props
+## Props
+
+This component offers the following props mentioned in the table below:
 
 <Props of={ToastContainer} />
 
 ## addToast
 
-Whenever you want to raise a toast, you can just call this function with the `variant` and `message` props. The other available parameters to customize your toast can be found under the Params title.
+This function can be called with the `variant` and `message` props whenever you want to raise a toast.
```

Comment:
Could you please add this line back `The other available parameters to customize your toast can be found under the Params title`

Reply:
ok